content/en/pod.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Pod
+status: Feedback Appreciated
+category: concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## What it is
+
+In [cloud native computing](/cloud-computing/), a "pod" is a basic building block for deploying and managing containerized applications. 
+A pod represents a single instance of an application, and can contain one or more [containers](/container/).
+[Kubernetes](/kubernetes/) manages pods as part of a larger deployment, and can scale pods up or down depending on the application's resource requirements.
```

Comment:
```suggestion
Kubernetes manages pods as part of a larger deployment and can scale pods up or down depending on the application's resource requirements.
```

Reply:
I would like to keep the reference to the kubernetes term. What I don't really like is that we say `... depending on the application's resource requirements` It might be true but when we use the word resource here, it kind of implies that we are referring to vertical scaling where we scale up resources of the pod itself rather than scaling horizontally. My suggestion is either to explicitly say: `...and can scale pods up and down (vertical and horizontal) ....` maybe also add the references to the respective terms.

Reply:
I removed the link to Kubernetes because I linked to it in the first sentence already. Updated the language based on your comment.